src/processors/modulation/Tremolo.cpp

```diff
@@ -170,13 +179,21 @@ void Tremolo::processAudio (AudioBuffer<float>& buffer)
         }
 
         // copy modulation data into all the channels
-        for (int ch = 1; ch < numChannels; ++ch)
+        for (int ch = 1; ch < numOutChannels; ++ch)
             audioOutBuffer.copyFrom (ch, 0, audioOutBuffer, 0, 0, numSamples);
 
         // multiply with incoming audio data
-        for (int ch = 0; ch < numChannels; ++ch)
+        for (int ch = 0; ch < numOutChannels; ++ch)
         {
-            const auto* x = audioInBuffer.getReadPointer (ch);
+            if (stereoMode && ch == 1)
+            {
+                // Flip phase on right LFO
+                auto* xOut = audioOutBuffer.getWritePointer (ch);
+                FloatVectorOperations::negate (xOut, xOut, numSamples);
+                FloatVectorOperations::add (xOut, 1.0f, numSamples);
```

Comment:
So this should work, but it might be better to do this in the loop on line 182-183, so we can save a copy operation? Does that make sense?

Reply:
Mmm, I'm not sure I understand why moving the operation would save a copy? Do we not need to copy either way?

Reply:
Right so basically we could change that for loop to look like:
```cpp
for (int ch = 1; ch < numOutChannels; ++ch)
{
    if (stereoMode)
    {
        // invert modulation signal (no need to copy)
        FloatVectorOperations::negate (audioOutBuffer.getWritePointer (ch), audioOutBuffer.getReadPointer (0), numSamples);
        FloatVectorOperations::add (audioOutBuffer.getWritePointer (ch), 1.0f, numSamples);
    }
    else
    {
        // just copy
        audioOutBuffer.copyFrom (ch, 0, audioOutBuffer, 0, 0, numSamples);
    }
}
```

Reply:
Ah I see, I guess I'm still confused how we're able to flip the modulation signal in the `audioOutBuffer`, when my understanding was that the modulation data was written into the `0` channel of the `audioOutBuffer` and  so the other channels of `audioOutBuffer` were empty?

Reply:
Oh nice! I see, we can use the read write pointers instead of just the write